code/modules/mob/living/silicon/robot/robot_model.dm

```diff
@@ -547,7 +547,7 @@
 	if(reagents.has_chemical_flag(REAGENT_CLEANS, 1))
 		our_turf.wash(CLEAN_SCRUB)
 
-	reagents.expose(our_turf, TOUCH, 10)
+	reagents.expose(our_turf, TOUCH, min(1, 10 / reagents.total_volume))
```

Comment:
![image](https://user-images.githubusercontent.com/58055496/162669796-a23b8b32-1d4e-4815-bbf5-07c33b72696a.png)

lmao lol said the scorpion

Reply:
mop code does this because it's only got like 2 u of the bucket's mix. We have way more then that, so there's absolutely no reason to multiply, maybe even at all honestly. I'm just bad at my job.

This is a fine fix.